src/RubyGosu.cxx

```diff
@@ -8376,6 +8376,34 @@ _wrap_Song_volumee___(int argc, VALUE *argv, VALUE self) {
 }
 
 
+SWIGINTERN VALUE
```

Comment:
This should include an upcall-branch to actually call the ruby function if available but don't know how to trigger it.